lighthouse-core/test/fraggle-rock/scenarios/api-test-pptr.js

```diff
@@ -4,21 +4,22 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import {jest} from '@jest/globals';
+import jestMock from 'jest-mock';
 
 import * as lighthouse from '../../../fraggle-rock/api.js';
 import {createTestState, getAuditsBreakdown} from './pptr-test-utils.js';
 import {LH_ROOT} from '../../../../root.js';
 
-jest.setTimeout(90_000);
+describe('Fraggle Rock API', function() {
+  // eslint-disable-next-line no-invalid-this
+  this.timeout(120_000);
```

Comment:
Why is the test declared like this, with the named function and timeout on `this`. It doesn't match what was done in other files.

Reply:
This method sets the timeout for all nested tests, adding `.timeout(...)` to the end of the describe block seems to only effect describe / the lifecycle methods.